apps/industrial-hemp/sections/summary-data-sections.js

```diff
@@ -160,6 +160,33 @@ module.exports = {
       {
         step: '/criminal-conviction',
         field: 'has-anyone-received-criminal-conviction'
+      },
+      {
+        step: '/other-regulatory-licences',
+        field: 'hold-other-regulatory-licences'
+      },
+      {
+        step: '/other-licence-details',
+        field: 'other-licence-details',
+        parse: (list, req) => {
+          if(req.sessionModel.get('hold-other-regulatory-licences') === 'no') {
+            return null;
+          }
```

Comment:
this can be simplified by using `dependsOn` instead

```suggestion
        dependsOn: 'hold-other-regulatory-licences',
        parse: (list, req) => {
```